Jenkinsfile.groovy

```diff
@@ -272,7 +272,7 @@ pipeline {
 
           sh "pnpm install --lockfile-only"
           sh "git add pnpm-lock.yaml"
-          sh "git commit --amend --no-edit"
+          sh "git commit -m \"chore(release): [version bump] - v${NEW_VERSION}\""
```

Comment:
The previously added `--no-git-tag-version` option of the `lerna publish` command doesn't create a commit of the bumped files (undocumented behavior). So instead of amending the previous commit we need to create a new one here.